### PR TITLE
Implementing yaml migration tool for schedule

### LINKED ISSUE #1821: Implement migration script from `scheduler: ""` to `Schedule` object
There is no more `scheduler` field in the specification of `*Chaos` object, so we need to add a script to migration from `scheduler:""` to `Schedule` object.

### LINKED ISSUE #1887: modify examples for nirvana
Modify the resources under `examples` directory for nirvana.

### PR DESCRIPTION
**Pull request by AsterNighT**
Signed-off-by: AsterNighT <klxjt99@outlook.com>

### What problem does this PR solve?

Fix #1821
Fix #1887
The workflow example is not yet changed. 
### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Breaking backward compatibility

Related changes

- [ ] Need to update the documentation

### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Comment by AsterNighT** [comment]
A problem is that it removes the quotes around strings. I'm pretty not sure whether it is good or not.

**Comment by AsterNighT** [comment]
I think it OK now. Referring to the document it seems proper not to enclose strings in quotes if no escape is needed,

**Comment by AsterNighT** [comment]
@YangKeao PTAL

File: examples/etcd/etcd-iochaos.yaml [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-  percent: 50
-  duration: "400s"
-  scheduler:
-    cron: "@every 10m"
-
+  schedule: '@every 10m'
+  type: IoChaos
+  historyLimit: 5
+  concurrencyPolicy: Forbid
+  io_chaos:
```
Comment (fewdan) [inline examples/etcd/etcd-iochaos.yaml:10]: According to the latest CRD definition, it should be "ioChaos"?

File: examples/jvm/jvm-cfl-example.yaml [hunk 2]
```diff
@@ -1,15 +1,18 @@
...
-    labelSelectors:
-      app: springboot-jvmchaos-demo
-  duration: "30s"
-  scheduler:
-    cron: "@every 1m"
+  schedule: '@every 1m'
+  type: JVMChaos
+  historyLimit: 5
+  concurrencyPolicy: Forbid
+  jvm_chaos:
```
Comment (fewdan) [inline examples/jvm/jvm-cfl-example.yaml:11]: change  "jvm_chaos"  to "jVMChaos" ?    There are similar situations in the following yaml files, so I will not list them one by one.

**Pull request closed without merging**